projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-test.properties

```diff
@@ -78,3 +78,8 @@ datajobs.executions.logsUrl.dateFormat=unix
 # Python versions supported for the purposes of the deployment integration tests
 datajobs.deployment.supportedPythonVersions={3.9: {vdkImage: 'registry.hub.docker.com/versatiledatakit/quickstart-vdk:release', baseImage: 'versatiledatakit/data-job-base-python-3.9:latest'}, 3.8: {vdkImage: 'registry.hub.docker.com/versatiledatakit/quickstart-vdk:pre-release', baseImage: 'versatiledatakit/data-job-base-python-3.8:latest'}}
 datajobs.deployment.defaultPythonVersion=3.9
+
+# Needed to run TestJobDeployTempCredsIt. If not using IAM user the assumeIAMRole in the test's
+# property should be changed to - datajobs.aws.assumeIAMRole=false and theese can be left empty:
```

Comment:
nit: may explicitly add to the section `datajobs.aws.assumeIAMRole=true`

Reply:
No, since all other tests run with this setting set to false by default.